reframe/frontend/cli.py

```diff
@@ -1,21 +1,24 @@
 import os
 import socket
 import sys
+import traceback
 
 import reframe
 import reframe.core.logging as logging
 import reframe.utility.os as os_ext
-from reframe.core.exceptions import (EnvironError, ReframeError,
+
```

Comment:
Please remove this blank line. The correct grouping of imports is the following:

```python
import standard_library_stuff
from standard_library_stuff import names

import reframe_stuff
from reframe_stuff import rfm_names
```

The order of the statements for each subgroup is first the `import` statements, then the `from .. import ..` ones, and each of these subsubgroups *must* be sorted alphabetically.